website/docs/10_going-deep/instrumentation.md

```diff
@@ -0,0 +1,138 @@
+# Booster instrumentation
+
+## Trace Decorator
+The Trace Decorator is a **Booster** functionality that facilitates the reception of notifications whenever significant events occur in Booster's core, such as event dispatching or migration execution.
+
+### Usage
+To configure a custom tracer, you need to define an object with two methods: onStart and onEnd. The onStart method is called before the traced method is invoked, and the onEnd method is called after the method completes. Both methods receive a TraceInfo object, which contains information about the traced method and its arguments.
```

Comment:
You eventually called the type `TraceParameters`, instead of TraceInfo (which, BTW, I like more)

Reply:
ok :)